src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/SampleFiltering/components/LineageFilter/index.tsx

```diff
@@ -219,44 +160,18 @@ export function LineageFilter({
     const newSelectedLineages = newSelectedOptions.map(
       (option: DefaultMenuSelectOption) => option.name
     );
-    // What we actually emit as selection does not always match user selected.
-    let emittedSelection: string[];
 
     // When beginning selection process, had nothing selected / "All" selected
```

Comment:
Nit: should probably remove the `/ "All" selected` clause of comment here since we don't have "All" as a real option and it's just what we display when 0 length of options chosen.